sc_wgs_monitoring/utils.py

```
@@ -27,6 +28,49 @@ def load_config(config_file) -> Dict:
+        slack_alert_channel = os.environ.get("SLACK_ALERT_CHANNEL")
+        sc_wgs_db = os.environ.get("DB_NAME")
+        postgres_user = os.environ.get("DB_USER")
+        postgres_pwd = os.environ.get("DB_PASSWORD")
+
+    except KeyError as e:
+        key = e.args[0]
+
+        raise KeyError(
+            f"Unable to import {key} from environment, is an .env file "
+            "present or env variables set?"
+        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)